src/Powercord/plugins/pc-sdk/components/ForceUI.jsx

```diff
@@ -49,7 +49,7 @@ class ForceUI extends React.PureComponent {
       const discordTextarea = document.querySelector('form > div > div > div');
       const instance = getOwnerInstance(discordTextarea);
 
-      const ChannelTextAreaForm = instance?._reactInternals?.return?.return;
+      const ChannelTextAreaForm = instance?._reactInternals?.return?.return?.return?.return?.return;
```

Comment:
Why not use `findInReactTree`? This is bound to break, just like the old `children.props[0].children[2].props.children.props` garbage that is still everywhere.

Reply:
This was resolved over Discord.
![image](https://user-images.githubusercontent.com/54973868/181622782-282ebcc9-5ad9-4fe0-bfa0-6e567b8b4a32.png)